agent/custom/general_navigation_action.py

```
@@ -2,9 +2,99 @@
 """
 通用导航 Action - 纯滑动 + YOLO检测 + 对比识别 + 特殊关卡ROI
 
-日月前事
+地月后事
 
-【鸽子衔枝之年】
+【雷夫衔枝之年】
+
+天上永恒的王座到来，人类之理为之焕然一新。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)